### PR TITLE
Milestone 3: Events

### PR DESCRIPTION
**Pull request by dqarias**
In this milestone we:
- copy the files calculate.js and operate.js into a logic/ directory in your project.
- use event handlers, state and props, pass props to child components, use lifecycle methods, lift state up

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 /* eslint-disable react/prefer-stateless-function */
+/* eslint-disable import/no-extraneous-dependencies */
+
```
Comment (ahmetbozaci) [inline src/components/Button.js]: - [ ] Kindly don't disable linters. If is giving errors please fix them.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @dqarias  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- Calculator working well without any error. ✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
- [ ] Kindly fill the acknowledgment section in a proper way to make a professional readme.
- [ ] Please fix the broken link in the license section to make a professional readme.
image
_Readme is not in files changed so ı didn't give an inline comment._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: src/components/Button.js (hunk 1 above)
Comment (dqarias) [reply src/components/Button.js]: Thanks, I've just done the required changes.

**Review by OLIPLICHE — APPROVED** [review]
Hi @dqarias , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**